Formula/ship.rb

```diff
@@ -15,6 +15,9 @@ class Ship < Formula
     sha256 cellar: :any_skip_relocation, mojave:         "c3974dea38bf106223fc9bccb8c3e2eaff6f8d951a95ddad849a63edc6040578"
   end
 
+  # depends indirectly on python@2 and is superseded by kots
+  deprecate! date: "2019-11-22", because: :deprecated_upstream
```

Comment:
Let's use today's date. That will make it easier to work out when this can be disabled.